doc/migration/version-upgrade-guide.md

```diff
@@ -486,6 +486,75 @@ All method signatures for [`sigs.k8s.io/controller-runtime/pkg/client.Client`](h
     - Delete the old CRD manifest files.
     - Rename CR manifest file names from `<group>_<version>_<kind>_cr.yaml` to `<full group>_<version>_<kind>_cr.yaml`.
 
+## `v0.12.x`
+
+### Go version
+
+- Ensure that you are using a go version 1.13+
+
+### modules
```

Comment:
We need more detail about dependencies because we no longer support `dep`/`Gopkg.toml`. We should tell users how to [migrate to Go modules](https://blog.golang.org/migrating-to-go-modules) if they are still using `dep`.